feat(aps): add troubleshooting apple id (#3877)
scaleway/docs-content: bare-metal/apple-silicon/troubleshooting/cant-create-apple-account.mdx

---
meta:

>   title: Troubleshooting account creation for hosted Mac mini's

Reviewer: ```suggestion
  title: Troubleshooting account creation for hosted Mac minis
```

  description: This page suggests solutions for when you cannot create an Apple Account directly from your hosted Mac mini
content:

>   h1: Troubleshooting account creation for hosted Mac mini's

Reviewer: ```suggestion
  h1: Troubleshooting account creation for hosted Mac minis
```

>   paragraph: This page suggests solutions for when you cannot establish a remote desktop connection to your Mac mini

Author: ```suggestion
  paragraph:  This page suggests solutions for when you cannot create an Apple Account directly from your hosted Mac mini
```

### Creating an Apple Account via the Apple website

If you are unable to create an Apple Account on your hosted Mac mini, follow these steps to create one through the Apple website:

1. Open your web browser and navigate to the Apple Account creation page: [https://account.apple.com/account](https://account.apple.com/account).
2. Click **Create Your Apple Account**. You will be redirected to a page where you can start the registration process.
3. Fill in your personal information:
   - First and Last Name: Enter your full name.
   - Country/Region: Select your country or region from the dropdown menu.
   - Birthdate: Enter your date of birth.
   - Email Address: Enter your existing email address. This will be your new Apple Account.
   - Password: Create a strong and secure password for your account.
   - Phone number: Enter your phone number. It will be used for two factor authentication and account recovery.
   Verify your information and click **Continue** to proceed.
4. Apple will send a verification email to the address you provided. Open the email and follow the instructions to verify your account.

> 

Author: ```suggestion
```

5. After verification, you will need to agree to Apple's terms and conditions to complete the process. Tick the checkbox and click on the corresponding button to agree.

> 

Author: ```suggestion
```

> 6. After successfully creating your Apple Account, return to your device and sign in using your new credentials.

Author: ```suggestion
6. After successfully creating your Apple Account, return to your hosted Mac mini and sign in using your new credentials.
```
